templates/reviewer-approval.pt.md

```diff
@@ -0,0 +1,11 @@
+
+## Resposta do revisor
+
+#### Aprovação final (pós-revisão)
+
+- [ ] **O autor respondeu à minha revisão e realizou as mudanças requisitadas. Eu recomendo a aprovação deste pacote.**
```

Comment:
The same for autor?  (I'm sorry I don't know if autor is masculine only).

Reply:
Yes!